relay-base-schema/src/metrics.rs

```diff
@@ -25,7 +25,7 @@ pub fn is_valid_metric_name(name: &str) -> bool {
 /// measurements.
 ///
 /// Units and their precisions are uniquely represented by a string identifier.
-#[derive(Clone, Copy, Debug, Eq, PartialEq, Hash, Default)]
+#[derive(Clone, Copy, Debug, Eq, PartialEq, Hash, Default, PartialOrd)]
```

Comment:
Q: what do we need these `PartialOrd` for?

Reply:
oh damn, i needed them for a different way i tried of looking for unique elements, but currently i dont need, thanks for pointing this out!